core/local/steps/add_infos.js

```diff
@@ -24,10 +30,23 @@ module.exports = function (buffer /*: Buffer */, opts /*: { syncPath: string } *
           event._id = id(event.path)
           if (['created', 'modified', 'renamed'].includes(event.action)) {
             log.debug({path: event.path, action: event.action}, 'stat')
-            event.stats = await fse.stat(path.join(opts.syncPath, event.path))
+            if (winfs) {
+              // XXX It would be better to avoid sync IO operations, but
+              // before node 10.5.0, it's our only choice for reliable fileIDs
```

Comment:
`FIXME: upgrade node to v10.5.0`?

Reply:
Well, currently, there is no version of electron with node v10.5.0+

Reply:
The goal is not to forget to do it when we can